Library/Homebrew/test/dev-cmd/bottle_spec.rb

```diff
@@ -415,6 +415,90 @@ def install
 
 
 
+          # something here
+
+        end
+      EOS
+    end
+  end
+
+  describe "--merge with --root-url-specs", :integration_test do
```

Comment:
Would rather not add another integration test for this. They are really slow and this is very niche behaviour.

Reply:
Makes sense! Doesn’t look like there are any direct tests of ‘bottle_output’ at present. Would you suggest adding some or not bothering with tests here?